src/contracts/proxies/ODSafeManager.sol

```diff
@@ -158,6 +162,8 @@ contract ODSafeManager is IODSafeManager {
     SAFEData memory _sData = _safeData[_safe];
     if (_dst == _sData.owner) revert AlreadySafeOwner();
 
+    _safeData[_safe].nonce += 1;
```

Comment:
Good. Are there any other functions we would wish to iterate the nonce to reset permissions?

Reply:
No, this should only happen on transfer of the ownership so that permissions previously granted are reset even if sent back to the original owner.